includes/class-payment-gateways.php

```diff
@@ -138,12 +138,6 @@ public function _admin_warnings() {
             }
         }
 
-        if ( ! $at_least_one_gateway ) {
```

Comment:
Was there a reason to remove this message?

Reply:
With the option to turn payment on/off, the message didn't make sense. Looking at it now, it will need some text change. I will add some

Reply:
Removed the message from the Fee Plan section. Not sure if we should keep it or change the message. It feels redundant showing the same message in the fee plans page.

Reply:
I made some adjustments here to show each of these messages when it make more sense.